### PR TITLE
Update GitHub workflow to use Windows 2022

### PR DESCRIPTION
**Pull request by chenilim**
#### Summary
Use windows-22 for GitHub build

### CODE REVIEW

**Comment by chenilim** [comment]
Meant this to be a draft. I updated .NET to 4.8 (from 4.7) to see if that fixes the Windows 2022 dependencies.

**Comment by chenilim** [comment]
Ok, after updating the Windows 10 SDK path, this passes the GitHub build with the Windows Server 2022 image. The new requirements are: .NET 4.8 and Windows 10 SDK 10.0.19041.0 (added to readme).

**Comment by harshilsharma63** [comment]
@chenilim does this affect the OS/system requirements for running the Boards desktop app?

**Comment by chenilim** [comment]
@harshilsharma63, yes .NET 4.8 is now required (up from 4.7.2). [Documentation](https://docs.microsoft.com/en-us/dotnet/framework/migration-guide/version-compatibility). However, the Windows SDK itself doesn't affect the version of Windows required (as long as new APIs aren't used).

File: README.md [hunk 1]
```diff
@@ -84,7 +84,7 @@ You can build standalone apps that package the server to run locally against SQL
     * Open a git-bash prompt
     * `make win-wpf-app`
     * run `cd win-wpf/msix && focalboard.exe`
-    * *Requires: Windows 10*
+    * *Requires: Windows 10, .NET 4.8 developer pack, Windows 10 SDK 10.0.19041.0*
```
Comment (wiggin77) [inline README.md:87]: Nit:  would be handy to include a link to the SDK downloads page.

**Review by wiggin77 — APPROVED** [review]

**Review by sbishel — APPROVED** [review]

**Pull request merged**